filebeat/docs/fields.asciidoc

```diff
@@ -4895,7 +4895,7 @@ type: keyword
 --
 Confidence level determined.
 
-type: keyword
+type: integer
```

Comment:
Is is this already a number in the event produced by the beat? Looks like it should be since it comes from deviceFlexNumber1 which is a long from the cef processor.